docs/source/_toctree.yml

```diff
@@ -1,14 +1,10 @@
 - sections: 
   - local: index
     title: 🤗 Transformers
-  - local: quicktour
-    title: Quick tour
   - local: installation
     title: Installation
-  - local: philosophy
-    title: Philosophy
```

Comment:
Why are we removing this one? I thought the plan was to move it only?

Reply:
Sorry, got a little ahead of myself! 😅 Adding it back in, and we can move it to the Concept Guides once that is ready.